packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.tsx

```diff
@@ -459,6 +459,7 @@ export class ContextualMenu extends BaseComponent<IContextualMenuProps, IContext
 
     if (ev.which === openKey) {
       this._onItemSubMenuExpand(item, ev.currentTarget as HTMLElement);
+      ev.preventDefault();
```

Comment:
I think it would be probably make the most sense to add it. It would at least be consistent with other places where we handle events.

I'd be interested in how it's cleaner. It's not obvious to me, I might be naive here tho! I thought about the purposes of the 2 methods:

`preventDefault`: Prevents the default *browser behavior*. It is intended to prevent things like scrollbars moving, buttons submitting forms, and anchors from navigating.

`stopPropagation`: Prevents the event from bubbling, which avoid *user code*. It doesn't affect browser behaviors; it simply avoids user code from executing. This is important when an event has been handled and it is intended for no other actions to handle the event.

So interpretting `defaultPrevented` as a signal for avoiding *user code* might be presumptuous or wrong.

Again, possible to watch for `defaultPrevented`, it just may not be the right signal indicating that user code executed an action and subsequent user code actions should be avoided.